service/ec2/customizations.go

```diff
@@ -4,19 +4,41 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
-	request "github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/internal/awsutil"
 )
 
+const (
+	// customRetryerMaxNumRetries sets max number of retries
+	customRetryerMaxNumRetries = 3
+
+	// customRetryerMinRetryDelay sets min retry delay
+	customRetryerMinRetryDelay = 1 * time.Second
+
+	// customRetryerMaxRetryDelay sets max retry delay
+	customRetryerMaxRetryDelay = 8 * time.Second
+)
+
+// setCustomRetryer overides the default Retryer values
+func setCustomRetryer(d *aws.DefaultRetryer) {
```

Comment:
minor nit, this is configuring the default retryer's options not setting the custom retryer. Maybe changed to `setRetryerConfig`?